streams/src/main/java/org/apache/kafka/streams/processor/internals/AssignedTasks.java

```diff
@@ -56,9 +55,12 @@
 
     AssignedTasks(final String logPrefix,
                   final String taskTypeName) {
-        this.logPrefix = logPrefix;
         this.taskTypeName = taskTypeName;
 
+        final LogContext logContext = new LogContext(logPrefix);
```

Comment:
I think this `LogContext` should be passed in? It should be the same as created for `StreamThread`

Reply:
I wanted to keep the same method signature but passing the `LogContext` makes sense.